src/quantization.cpp

```diff
@@ -57,6 +58,7 @@ void quantize_fp16(program& prog, const std::vector<std::string>& ins_names)
 {
     run_passes(prog,
                {normalize_ops{},
+                rewrite_low_precision{},
```

Comment:
This pass should run after `rewrite_reduce`.

Reply:
Thanks! That makes more sense.